integrations/zoom.js

```diff
@@ -15,6 +20,11 @@ module.exports = async function(robot, kredits) {
 
     return getContributorByZoomDisplayName(displayName)
       .then(contributor => {
+        if (!contributor) {
+          robot.logger.error(`[hubot-kredits] Contributor not found: Zoom display name: ${displayName}`);
+          messageRoom(`I tried to add a contribution for zoom user ${displayName}, but did not find a matchig contributor profile.`);
```

Comment:
I think we should not do this. It will be a normal thing in a public call that not everyone is a contributor with a profile.